src/db/inout/exportcsv.cc

```diff
@@ -612,8 +641,15 @@ void export_asset_json(std::ostream& out, std::set<std::string>* listElements)
 
                 cxxtools::SerializationInfo& si_outlet_label = si_outlet_x.addMember("");
                 si_outlet_label.addMember("name") <<= "label";
-                si_outlet_label.addMember("value") <<= oneOutlet.second.label;
-                si_outlet_label.addMember("read_only") <<= oneOutlet.second.label_r;
+                // if label is empty, get index value
```

Comment:
comment shall be updated

Reply:
comment still seem to be valid. Though I may stand corrected by missing something

Reply:
ok